modules/ble/src/main/resources/META-INF/substrate/dalvik/AndroidManifest.xml

```diff
@@ -3,6 +3,7 @@
     <uses-sdk android:minSdkVersion="21" />
     <uses-permission android:name="android.permission.BLUETOOTH" />
     <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
-    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.BLUETOOTH_SCAN" />
```

Comment:
Do you need to add `android:maxSdkVersion="30"` to legacy permissions like in [here](https://developer.android.com/guide/topics/connectivity/bluetooth/permissions#declare-android12-or-higher)?